OpenSubtitlesHandler/Util.cs

```diff
@@ -0,0 +1,152 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Text;
+using System.Text.Json;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace OpenSubtitlesHandler
+{
+    public static class Util
+    {
+        private static readonly HttpClient HttpClient = new HttpClient();
+        private static readonly JsonSerializerOptions SerializerOpts = new JsonSerializerOptions { IncludeFields = true, PropertyNamingPolicy = SnakeCaseNamingPolicy.Instance };
+        public static Action<string> OnHttpUpdate = _ => {};
+        private static string _version = string.Empty;
+
+        public static DateTime NextReset
```

Comment:
OpenSubtitles changed when download limits are reset, this method is not correct anymore

Reply:
does that mean that more changes would be required to this PR or is a followup required? (sorry for the prolonged waiting times)

Reply:
This PR, I'm just waiting to get a clarification from the devs of opensubtitles

Reply:
Rodger that, thank you for the heads up!

Reply:
solved in a7f94b5bb3950da785f1cf75c2309c89b49d6f80